annotation-processor/src/main/java/org/hibernate/validator/ap/util/ConstraintHelper.java

```diff
@@ -518,7 +674,7 @@ else if ( assignableTypes.size() == 1 ||
 	 * @return A set with the supported types.
 	 */
 	private Set<TypeMirror> getSupportedTypes(DeclaredType constraintAnnotationType) {
-
+		
```

Comment:
Better add not the trailing whitespace (also some lines below). Actually I'd expect CS to fail due to that, but maybe I am dreaming. You can add it as a "Save action" in Eclipse to automatically remove any trailing whitespace. That setting works great for me.